packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -464,6 +464,22 @@ const initializeEditor = (editorInstance: any) => {
 	editor = editorInstance;
 };
 
+const handleConfigModelResp = async (resp: TaskResponse, type: ClientEventType) => {
+	const taskIdRefs = {
+		[ClientEventType.TaskGollmConfigureModel]: modelFromDocumentTaskId,
+		[ClientEventType.TaskGollmConfigureFromDataset]: modelFromDatasetTaskId
+	};
+	taskIdRefs[type].value = [TaskStatus.Success, TaskStatus.Cancelled, TaskStatus.Failed].includes(
+		resp.status
+	)
+		? ''
+		: resp.id;
```

Comment:
What is this use for?

Reply:
When a task is finished, it sets the task Id to empty string so that we know the task is finished, otherwise it's set to running task id. I just did more cleanups and simplified this in latest commit.